README.md

```diff
@@ -72,6 +76,8 @@ If you are unfamiliar with Grunt you may want to head over to
 
 
 ### Verify
+After installing beaker, make sure to `npm install` in order to get the packages in the `package.json` placed by Beaker.
```

Comment:
Actually, it should probably be `make install` since at least one of our current dependencies needs extra love after the `npm install` which is encapsulated in `make install`